libs/barista-components/tag/src/tag-add/tag-add.html

```diff
@@ -5,9 +5,10 @@
     cdk-overlay-origin
     (click)="open()"
     #origin="cdkOverlayOrigin"
+    [title]="buttonLabel ?? title"
```

Comment:
```suggestion
```
You don't need to provide a title for the button if you provide a text for it :)

Reply:
Now I see the dilemma, thanks for the clarification. Let's keep that title attribute then on the button.